blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IOContainer.java

```diff
@@ -34,6 +38,9 @@
     private static final Configuration configuration = Configuration.getInstance();
 
     private ShardStateIO shardStateIO;
+    private MetadataIO metadataIO;
+    private TestIO testIO;
```

Comment:
Yeah, I was thinking about that as well, but was curious if other tests might have similar test helper methods.  I'll move it into integration